lib/ansible/modules/windows/win_scheduled_task.ps1

```diff
@@ -73,9 +116,18 @@ try {
         $exists = $true
     }
     elseif ( ($measure.count -eq 0) -and ($state -eq "absent") ){
-        # Nothing to do
+        # Nothing to do for the task
         $result.exists = $false
         $result.msg = "Task does not exist"
+
+        # Remove task path if it exists
```

Comment:
I would not do it like this. Since we are managing tasks, I would only remove the TaskPath when the task was actually removed (and we are already making changes). If there was no task te remove, I would leave the TaskPath as-is, because it's unlikely we created the TaskPath anyhow.